### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by tiffany-taylor**
This should resolve the build. Tested locally.

### CODE REVIEW

File: reference/datetime/datetime/settime.xml [hunk 1]
```diff
@@ -102,7 +102,7 @@
       <entry>The behaviour with double existing hours (during the fall-back
       DST transition) changed. Previously PHP would pick the second occurrence
       (after the DST transition), instead of the first occurrence (before DST
-      transition).
+       transition).</entry>
```
Comment (salathe) [inline reference/datetime/datetime/settime.xml:105]: Nit: added an unnecessary space character before `transition`

File: reference/datetime/datetimeimmutable/settime.xml [hunk 2]
```diff
@@ -81,7 +81,7 @@
       <entry>The behaviour with double existing hours (during the fall-back
       DST transition) changed. Previously PHP would pick the second occurrence
       (after the DST transition), instead of the first occurrence (before DST
-      transition).
+       transition).</entry>
```
Comment (salathe) [inline reference/datetime/datetimeimmutable/settime.xml:84]: Nit: again, added an unnecessary space character before `transition`

File: reference/datetime/datetimeimmutable/construct.xml [hunk 3]
```diff
@@ -211,7 +211,6 @@ echo $time->format('Y/m/d H:i:s'), "\n";
 ]]>
    </screen>
   </example>
- </refsect1>
```
Comment (salathe) [inline reference/datetime/datetimeimmutable/construct.xml:214]: Good. :)

**Review by salathe — COMMENTED** [review]
Thanks for fixing the build! 👍

File: reference/datetime/datetimeimmutable/settime.xml (hunk 2 above)
Comment (tiffany-taylor) [reply reference/datetime/datetimeimmutable/settime.xml:84]: Should I push a new commit with these changes?

Comment (cmb69) [reply reference/datetime/datetimeimmutable/settime.xml:84]: Yes, please. If the changes are irrelevant for the translations (as in this case), please start the commit message with `[skip-revcheck]`.

**Pull request merged**